### ISSUE TITLE
plot_pdf norm=True option

### ISSUE THREAD

**Issue by anetasie** [sherpa/sherpa#886]
I'm looking at `plot_pdf` generated with `norm=True` option and it does not look right. The y-axis values are definitely larger than 1. It seems that the range of the parameter values are included in the normalization, so for the amplitude parameter of order 1e-4 the y axis shows numbers of 10000. . In the case of `norm=True` I expected the y-axis to be all `<1`.

**Comment by DougBurke** [comment]
@olaurino did some work on this in #647 - which was needed because of a warning that NumPy was creating. I don't know if this is related.

**Comment by DougBurke** [comment]
At the moment the y label is always 'probability density' when `norm=True` and `norm=False`. One of these should probably be changed...

**Comment by DougBurke** [comment]
Note that the parameter name is `normed` and not `norm` (it should error out if you use `norm`).

**Comment by anetasie** [comment]
@DougBurke  yes, I made a typo. It should be `normed=True`. There is some change in the normalization between `normed=True` and `normed=False` but the plot is not correct.
